examples/pulse.py

```diff
@@ -140,22 +137,16 @@ def main(actx_class, use_esdg=False,
         generate_mesh = partial(generate_regular_rect_mesh,
             a=(box_ll,)*dim, b=(box_ur,)*dim,
             nelements_per_axis=(nel_1d,)*dim,
-            boundary_tag_to_face={
-                "outlet_L": ["+y"],
-                "outlet_R": ["-y", "+x"],
-                "inlet": ["-x"]})
+            periodic=(True,)*dim)
```

Comment:
Can we make it have AdiabaticSlipWalls instead?

Reply:
https://github.com/illinois-ceesd/mirgecom/pull/970/commits/7cc91555139d7b27aaba6d7fee2d14dce230a3af